source/Irrlicht/CBAWMeshWriter.h

```diff
@@ -21,11 +23,42 @@ namespace scene
 
 	class CBAWMeshWriter : public IMeshWriter
 	{
+	public:
+		enum E_COMPRESSION_TARGETS
+		{
+			ECT_NOTHING = 0x00,
+			ECT_RAW_BUFFERS = 0x01,
+			ECT_TEXTURES = 0x02,
```

Comment:
what you use now for textures should really be ECT_TEXTURE_PATHS
Although ECT_TEXTURES should already be reserved